accounts/serializers/connection.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from accounts.models import connection
```

Comment:
- it's better to import the model from the its files directly not from the `models` module
- like: `from accounts.models.connection import Connection`